.env.example

```
@@ -1,14 +1,10 @@
-GROK_MODEL=grok-4-latest
-GROK_ENDPOINT=https://api.x.ai/v1
-GROK_RPS=1
-GROK_RPM=60
-GROK_MAX_RETRIES=3
-GROK_BACKOFF_MS=300
+LLM_PROVIDER=perplexity
+
+# === Perplexity API ===
+PERPLEXITY_API_KEY=__replace_me__
+PERPLEXITY_MODEL=sonar
+PERPLEXITY_ENDPOINT=https://api.perplexity.ai/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PERPLEXITY_ENDPOINT key should go before the PERPLEXITY_MODEL key

(UnorderedKey)